frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -207,37 +216,34 @@ export const AddCapacityModal = (props: AddCapacityModalProps) => {
           />
         ) : (
           <>
-            <FormGroup
-              className="pf-u-py-sm"
-              fieldId="request-size"
-              id="requestSize__FormGroup"
-              label={t('ceph-storage-plugin~Raw Capacity')}
-              labelIcon={<FieldLevelHelp>{requestedCapacityTooltip(t)}</FieldLevelHelp>}
-            >
-              <TextInput
-                isDisabled
-                id="request-size"
-                className={classNames('pf-c-form-control', 'ceph-add-capacity__input')}
-                type="number"
-                name="requestSize"
-                value={osdSizeWithoutUnit}
-                aria-label="requestSize"
-                data-test-id="requestSize"
-              />
-              {provisionedCapacity && (
+            {provisionedCapacity ? (
+              <FormGroup
+                fieldId="request-size"
+                label={t('ceph-storage-plugin~Raw Capacity')}
+                labelIcon={<FieldLevelHelp>{requestedCapacityTooltip(t)}</FieldLevelHelp>}
+                helperText={currentlyUsed}
+              >
+                <TextInput
+                  isDisabled
+                  id="request-size"
+                  className={classNames('pf-c-form-control', 'ceph-add-capacity__input')}
+                  type="number"
+                  name="requestSize"
+                  value={osdSizeWithoutUnit}
+                  aria-label="requestSize"
+                  data-test-id="requestSize"
+                />
                 <TextContent className="ceph-add-capacity__provisioned-capacity">
                   {' '}
                   {t('ceph-storage-plugin~x {{ replica, number }} replicas =', {
                     replica,
                   })}{' '}
                   <strong data-test="provisioned-capacity">{provisionedCapacity}&nbsp;TiB</strong>
                 </TextContent>
-              )}
-              <TextContent className="pf-u-font-weight-bold pf-u-secondary-color-100 ceph-add-capacity__current-capacity">
-                {t('ceph-storage-plugin~Currently Used:')}&nbsp;
-                {currentCapacity}
-              </TextContent>
-            </FormGroup>
+              </FormGroup>
+            ) : (
+              <>{currentlyUsed}</>
+            )}
```

Comment:
We can just do:
```
    {osdSizeWithoutUnit && <RawCapacity t={t} replica={replica} osdSizeWithoutUnit={osdSizeWithoutUnit} />}
     <CurrentlyUsedCapacity t={t} currentCapacity={currentCapacity} />
```
```
const CurrentlyUsedCapacity: React.FC<{ currentCapacity: any, t: any }> = ({ currentCapacity, t }) => (
  <TextContent className="pf-u-font-weight-bold pf-u-secondary-color-100 ceph-add-capacity__current-capacity">
    {t('ceph-storage-plugin~Currently Used:')}&nbsp;
    {currentCapacity}
  </TextContent>
);

const RawCapacity: React.FC<{ t: any, osdSizeWithoutUnit: any, replica: any, }> = ({ t, osdSizeWithoutUnit, replica }) => {
  const provisionedCapacity = getProvisionedCapacity(osdSizeWithoutUnit);
  return <>
    <FormGroup
      fieldId="request-size"
      label={t('ceph-storage-plugin~Raw Capacity')}
      labelIcon={<FieldLevelHelp>{requestedCapacityTooltip(t)}</FieldLevelHelp>}
    >
      <TextInput
        isDisabled
        id="request-size"
        className={classNames('pf-c-form-control', 'ceph-add-capacity__input')}
        type="number"
        name="requestSize"
        value={osdSizeWithoutUnit}
        aria-label="requestSize"
        data-test-id="requestSize"
      />
      <TextContent className="ceph-add-capacity__provisioned-capacity">
        {' '}
        {t('ceph-storage-plugin~x {{ replica, number }} replicas =', {
          replica,
        })}{' '}
        <strong data-test="provisioned-capacity">{provisionedCapacity}&nbsp;TiB</strong>
      </TextContent>
    </FormGroup>
  </>
};
```

This will be simpler and easy to follow.